doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+parameters can also be provided. In this case, we will be specifying the threshold
+used to find b=0 directions (``b0_threshold``) and the output directory (``out_dir``). 
+
+To get the mask file, run the command as::
+
+    dipy_median_otsu data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii --vol_idx 3 4 5 6 --out_dir "cfin_multib_mask"
+
+To run the DKI reconstruction method on the data, execute the ``dipy_fit_dki``
+command, e.g.::
+
+    dipy_fit_dki data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bval data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bvec cfin_multib_mask/brain_mask.nii.gz --b0_threshold 70.0 --out_dir recons_dki_output
+
+This command will save the DKI metrics to the specified output directory.
+
+--------------------
+Constant Solid Angle
+--------------------
+
+We are using ``stanford_hardi`` dataset. As with any other workflow in DIPY, you can
+also use your own data!
+
+Create a directory in which to save the peaks volume (e.g.: ``recons_csa_output``)::
+
+    mkdir recons_csa_output
+
+The workflow for the CSA reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+arguments can also be provided. In this case, we will be specifying whether or
+not to save pam volumes as single nifti files (``extract_pam_values``) and the
+output directory (``out_dir``).
+
+To perform the CSA reconstruction, we will run the command as::
+
+    dipy_fit_csa data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec stanford_hardi_mask/brain_mask.nii.gz --extract_pam_values --out_dir "recons_csa_output"
+
+This command will save the CSA metrics to the specified output directory.
+
+-----------------------------------
+Intravoxel incoherent motion (IVIM)
+-----------------------------------
+
+The intravoxel incoherent motion (IVIM) model describes diffusion
+and perfusion in the signal acquired with a diffusion MRI sequence
+that contains multiple low b-values. The IVIM model can be understood
+as an adaptation of the work of Stejskal and Tanner [Stejskal65]_
+in biological tissue, and was proposed by Le Bihan [LeBihan84]_.
+The model assumes two compartments: a slow moving compartment,
+where particles diffuse in a Brownian fashion as a consequence of thermal
+energy, and a fast moving compartment (the vascular compartment), where
+blood moves as a consequence of a pressure gradient. In the first compartment,
+the diffusion coefficient is $\mathbf{D}$ while in the second compartment, a
+pseudo diffusion term $\mathbf{D^*}$ is introduced that describes the
+displacement of the blood elements in an assumed randomly laid out vascular
+network, at the macroscopic level. According to [LeBihan84]_,
+$\mathbf{D^*}$ is greater than $\mathbf{D}$.
+
+We are using ``ivim`` dataset. You can also use your own data.
```

Comment:
Same comment as above concerning the first sentence.

Reply:
Addressed in 4ce067c